test/amazonica/test/lambda.clj

```diff
@@ -15,10 +15,10 @@
     {:access-key (get creds "aws_access_key_id")
      :secret-key (get creds "aws_secret_access_key")}))
 
-(def role (-> #(.contains (:role-name %) "lambda")
-            (filter (:roles (iam/list-roles)))
-            first
-            :arn))
+(def role (delay (-> #(.contains (:role-name %) "lambda")
```

Comment:
Added this `delay` because I don't have the expected aws profile in my `~`, so loading this ns would break my tools.namespace `(refresh)` experience

(same for a couple other refinements in tests below)